arbitrator/arbutil/src/pricing.rs

```diff
@@ -6,6 +6,8 @@ use crate::{
     Bytes32,
 };
 
+use crate::evm::api::Ink;
```

Comment:
this doesnt exist in the code: https://github.com/OffchainLabs/nitro/blob/v3.6.7/arbitrator/arbutil/src/pricing.rs